### ISSUE TITLE
RDF fixes: identificationProtocol

### ISSUE THREAD

**Issue by nielsklazenga** [hiscom/hispid#118]
I have removed dwc:identificationProtocol from the RDF. This term does not exist.

**Comment by AaronWilton** [comment]
Given that the addition of a term was ratified in Hobart - shouldn't the removal also be ratified?  (and in the meantime a correction applied to remove the incorrect attribution to DWC) 

We also agreed in Hobart on a vocabulary for this term - which hasn't been removed.

(Note - this doesn't necessarily mean i am in favour of retaining the term)
(Note2 - there is a related ABCD term, but it is a free text field)

**Comment by nielsklazenga** [comment]
Yes, all correct (and all agreed). Must have overlooked it at the meeting, because it was also an RDF thing.
